apps/api/v2/src/app.module.ts

```diff
@@ -53,7 +55,17 @@ import { AppController } from "./app.controller";
 })
 export class AppModule implements NestModule {
   configure(consumer: MiddlewareConsumer): void {
-    consumer.apply(AppLoggerMiddleware).forRoutes("*");
-    consumer.apply(RewriterMiddleware).forRoutes("/");
+    consumer
+      .apply(RawBodyMiddleware)
```

Comment:
Stripe requires a Buffer/String for the Webhooks to be built. Since NestJS by default parses everything as JSON, this was not letting the event through. That's why this change was required.